server/src/permissions/index.ts

```diff
@@ -101,13 +23,20 @@ const rules = {
     })
     return Boolean(user && user.role == 'ADMIN')
   }),
-  isWriter: rule()(async (_, args, ctx: Context) => {
+  isSourceWriter: rule()(async (_, args, ctx: Context) => {
     // Get user id
     const userId = getUserId(ctx)
 
+    // Return true if the user is admin
+    const user = await ctx.photon.users.findOne({
```

Comment:
just a thought for later: multiplying the db calls in an acl middleware impacts dthe performance badly, now that we have redis set up, we should probably cache the logged in users in there so we have a fast way to access user details

Reply:
I'll write an issue for that!